src/lib.rs

```diff
@@ -9,54 +9,49 @@
 // "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
 
-//! # Snowplow Rust Tracker
+//! # Snowplow Tracker
```

Comment:
Is it worth using multi-line comments?

Reply:
The [rust style guide](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md#comments) advises to prefer line comments over block comments